src/content/get-started/transfer-a-sparsified-model/nlp-text-classification.mdx

```diff
@@ -98,7 +98,7 @@ $ sparseml.transformers.text_classification -h
 With the sparsified model successfully trained, it is time to export it for inference.
 The `sparseml.transformers.export_onnx` command is used to export the training graph to a performant inference one.
 After the command completes, a `model.onnx` file is created in `models/deployment` folder.
-It is now ready for deployment with the DeepSparse Engine.
+It is now ready for deployment with DeepSparse.
```

Comment:
Since DeepSparse describes the suite of components, are you saying one can deploy with the whole suite? Are all four components doing the deployment?